apps/zero-runtime-vite-app/src/App.tsx

```diff
@@ -30,6 +59,7 @@ const HalfWidth = styled.div({
   maxHeight: 100,
   padding: 20,
   border: '1px solid #ccc',
+  animation: [`${bounce} 1s ease infinite`, `${bounceAnim} 1s ease infinite`],
```

Comment:
Should this be just a string of `${bounce} 1s ease infinite, ${bounceAnim} 1s ease infinite`?

Reply:
Depends on what you want to achieve. Array values will be generated as media query css for the theme breakpoints.

Reply:
I see. This is just an example, right?

Reply:
Correct